networking/multiple_networks/configuring-additional-network.adoc

```diff
@@ -19,13 +19,13 @@ As a cluster administrator, you can configure an additional network for your clu
 [id="{context}_approaches-managing-additional-network"]
 == Approaches to managing an additional network
 
-You can manage the life cycle of an additional network by two approaches. Each approach is mutually exclusive and you can only use one approach for managing an additional network at a time. For either approach, the additional network is managed by a Container Network Interface (CNI) plugin that you configure.
+You can manage the lifecycle of an additional network in OpenShift by using one of two approaches: modifying the Cluster Network Operator (CNO) configuration or applying a YAML manifest. Each approach is mutually exclusive and you can only use one approach for managing an additional network at a time. For either approach, the additional network is managed by a Container Network Interface (CNI) plugin that you configure. The two different approaches are summarized here: 
 
```

Comment:
There's another advantage to using the CNO -- you can define net-attach-defs at install time via install-config.